config/initializers/action_mailer.rb

```diff
@@ -0,0 +1,4 @@
+ActionMailer::Base.configure do |config|
+  # Use https when creating links in emails
+  config.default_url_options = { protocol: 'https', host: Spree::Config[:site_url] }
```

Comment:
Ah, I think we actually want to only do this in staging and production. Test and development have very different settings for links, emails and the site url.

Reply:
yes, that's right. We don't want to mess with that.